docs/extensions.md

```diff
@@ -10,7 +10,8 @@ Below is a list of supported extensions with links to their documentation pages.
 | Authenticator - OIDC Extension            | [oidcauthextension](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.109.0/extension/oidcauthextension/README.md)                 |
 | BindPlane Extension                       | [bindplaneextension](../extension/bindplaneextension/README.md)                                                                                           |
 | File Storage Extension                    | [filestorage](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.109.0/extension/storage/filestorage/README.md)                     |
-| Health Check Extension                    | [healthcheckextension](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.109.0/extension/healthcheckextension/README.md)           |
+| Health Check Extension                    | [healthcheckextension](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/headerssetterextension/README.md)           |
+| Headers Setter Extension                    | [headerssetterextension](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.109.0/extension/healthcheckextension/README.md)           |
```

Comment:
Looks like the links here are reversed. I also think the Headers Setter extension should come first (alphabetically)